docfiles/docs.js

```diff
@@ -47,8 +54,21 @@ function searchSubmit(form) {
 }
 
 function setupSidebar() {
+    let togglesidebar = document.getElementById("togglesidebar");
+    togglesidebar.onkeydown = handleEnterKey;
+
+
     $('.ui.sidebar')
-        .sidebar({ dimPage: false })
+        .sidebar({ 
+            dimPage: false,
+            onShow: () => {
+                togglesidebar.setAttribute("aria-expanded", "true");
+                document.getElementsByClassName("sidebar").item(0).getElementsByClassName("focused").item(0).focus();
+            },
+            onHidden: () => {
```

Comment:
Right. It is supported from IE 11 since 2015 (not 2013) with the release of W10.